docs/articles/nunit/release-notes/Nunit4.0-MigrationGuide.md

```diff
@@ -96,7 +96,7 @@ Assert.That(actualText, Does.StartWith("42"), $"Expected '{actualText}' to start
 There are no code fixers for `FileAssert` and `DirectoryAssert`. They could be added, but we don't expect these to be
 used too much.
 
-#### Convert Classic Assert into NUnit 4.x equivalent
+#### Using Classic Asserts in NUnit 4.x
```

Comment:
The steps outline how to migrate existing code to the new naming.
If you don't like `Convert`, as a minimum it should be `Update`.

Reply:
That's a good idea, I like "Update" over "Convert" as it feels "lighter".

What are your thoughts on `Classic Asserts` vs `Classic-style Asserts` ? I'm trying to think of wording for this section which emphasizes to the reader that they can choose to stick with their chosen style of asserts if it is the classic style - even if it means they must update the referenced class.

Reply:
I suggest `Updating from Classic Asserts in NUnit 4.x`

And I suggest it should be "Classic Asserts". We're calling the whole idea of them classic, not just the style.